stripe/__init__.py

```diff
@@ -2,6 +2,7 @@
 from typing import Optional
 import sys as _sys
 import os
+import warnings
```

Comment:
Any particular reason to move?

Reply:
It was also used in `_ensure_default_http_client()`,  but I moved this import back to its original location.